docs/contributing/docs-templates.md

```diff
@@ -3,69 +3,82 @@ title: Docs Templates
+Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios.
 
 ### What should a guide article be about?
 
-We need guide articles to describe every task you can accomplish with Gatsby.
+We need guide articles to describe every concept and task you can accomplish with Gatsby.
+
+Each guide article should explain exactly one concept and that concept should be apparent from the article's title. [Overview guides](#reference-guide-overview) can list child pages to present multiple ways of getting a job done while limiting the scope of each individual article (e.g. Styling Your Site, Using Layout Components, Standard Global CSS Files, etc.)
 
-Each guide article should explain exactly one task and that task should be
-apparent from the article's title.
+### Near-perfect example of a reference guide
 
-#### What if I want to include multiple tasks and concepts in a guide article?
+[Linking Between Pages](/docs/linking-between-pages/)
 
-If you find yourself wanting to include multiple related topics in one article, consider splitting each into its own individual guide and referencing the other topics under sections called “Prerequisites” and/or "Other
-Resources" sections in the related guide articles.
+### What if I want to include multiple tasks and concepts in a guide article?
+
+If you find yourself wanting to include multiple related topics in one article, consider splitting each into its own individual guide and referencing the other topics under sections called “Prerequisites” and/or "Other Resources" sections in the related guide articles.
 
 It’s more ideal to have many articles that cover a broad range of technical topics rather than smashing too many topics into one article.
 
 If you find yourself wanting to teach the reader how to accomplish a series of related tasks, you might want to write a tutorial.
 
-#### When to write a guide vs. a tutorial?
+### When to write a reference guide vs. a tutorial, vs. a recipe?
 
-A guide explains a discrete task without the step-by-step context provided by a tutorial.
+[Reference guide articles](#reference-guide-template) cover discrete topics as documentation with links to other resources. A reference guide explains a task or concept without the step-by-step context provided by a tutorial or recipe.
 
-Guides cover the smallest possible topic, while tutorials can cover a series of related tasks that you string together.
+[Tutorials](#tutorial-template) guide users through a series of related tasks they can string together successfully, with minimal distractions or detours.
 
-#### How to choose a guide article topic?
+[Recipes](#recipe-template) are a happy medium between step-by-step tutorials and crawling the full reference guides, by providing step-by-step guidance for small, common tasks.
```

Comment:
I'm not super familiar w/ recipes; I think it sounds nice though! Maybe a short list of examples would be helpful for people who are new to them (would be easier than making a template too)